src/main/java/com/jvm_bloggers/entities/newsletter_issue/NewsletterIssueRepository.java

```diff
@@ -18,4 +20,8 @@
 
     List<NewsletterIssue> findAllByOrderByPublishedDateDesc();
 
+    @Query("select case when count(*) > 0 then true else false end "
+        + "from NewsletterIssue where issueNumber = :issueNumber")
+    boolean issueNumberExist(@Param("issueNumber") Long issueNumber);
```

Comment:
Projections looks good, but if I'm checking correctly (sorry, no access to IDE, and have no idea how to check it from code) our version of Spring Data should already provide this feature: https://jira.spring.io/browse/DATAJPA-851 that would enable us `existsByNewsletterIssueId(Long issueNumber)`